src/global.css

```diff
@@ -1,5 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Signika&display=swap');
 
+:root {
+  --font-family: "Signika", sans-serif;
+  --color-primary: #91c788;
+  --color-secondary: #ff8473;
+  --color-secondary-light: #FFC0B8;
+  --color: #7C7C7C;
```

Comment:
maybe a more specific name for the variable?

Reply:
thx! Which variable do you mean? color?